CHANGELOG.md

```diff
@@ -18,6 +18,8 @@
 * [FEATURE] Query Frontend: Add `cortex_query_fetched_series_total` and `cortex_query_fetched_chunks_bytes_total` per-user counters to expose the number of series and bytes fetched as part of queries. These metrics can be enabled with the `-frontend.query-stats-enabled` flag (or its respective YAML config option `query_stats_enabled`). #4343
 * [FEATURE] AlertManager: Add support for SNS Receiver. #4382
 * [FEATURE] Distributor: Add label `status` to metric `cortex_distributor_ingester_append_failures_total` #4442
+* [FEATURE] Queries: Added `present_over_time` PromQL function, also some TSDB optimisations. #4505
+* [FEATURE] Compactor: Blocks can now be marked as exempt from compaction, e.g. if they are corrupted. #4505
```

Comment:
This feature wasn't enabled in #4505. That is the `false` parameter passed to `compact.NewBucketCompactor` in  https://github.com/cortexproject/cortex/pull/4505/files#diff-adc81f6ffba52bc1871b0dac7c88237aa1cdea40e7691e35e834b40129ab723f